packages/eslint-config-olo/rules/react.js

```diff
@@ -40,7 +48,7 @@ module.exports = {
     'react/jsx-max-props-per-line': [0, { 'maximum': 1 }],
     // Prevent usage of .bind() and arrow functions in JSX props
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md
-    'react/jsx-no-bind': 2,
+    'react/jsx-no-bind': [2, { 'allowArrowFunctions': true }],
```

Comment:
This rule is really outdated, we should consider disabling it. It's also not applicable to hooks-based components since they create functions every render. 

https://reactjs.org/docs/faq-functions.html#is-it-ok-to-use-arrow-functions-in-render-methods

Reply:
Per our conversation this morning, this is a style preference, and where it would error prior if there was a `this.memberFunction.bind(this)` or `() => doSomething()` in a component callback property, the addition of `allowArrowFunctions` ensures that we can do the latter and still retain the safety of the reminder to bind a class method.